frontend/src/ui-components/AcmDropdown/AcmDropdown.tsx

```diff
@@ -170,7 +170,6 @@ export function AcmDropdown(props: AcmDropdownProps) {
           )
         }
         isOpen={isOpen}
-        isPlain={props.isPlain}
```

Comment:
@lennysgarage We can't remove this from the shared component AcmDropdown as it might affect other components using it. Can you find where in App UI we're using AcmDropdown for the create button and set the isPlain prop to `false`?

Reply:
Looks like it was being set in Overview.tsx, updated it now!